rbc/irtools.py

```diff
@@ -338,31 +380,87 @@ def compile_to_LLVM(functions_and_signatures,
         flags.set('no_cfunc_wrapper')
 
     function_names = []
+    succesful_fids = []
+    required_libraries = set()
     for func, signatures in functions_and_signatures:
-        for sig in signatures:
+        for fid, sig in signatures.items():
             fname = func.__name__ + sig.mangling()
-            function_names.append(fname)
             args, return_type = sigutils.normalize_signature(
                 sig.tonumba(bool_is_int8=True))
-            cres = compiler.compile_extra(typingctx=typing_context,
-                                          targetctx=target_context,
-                                          func=func,
-                                          args=args,
-                                          return_type=return_type,
-                                          flags=flags,
-                                          library=main_library,
-                                          locals={},
-                                          pipeline_class=pipeline_class)
+            try:
+                cres = compiler.compile_extra(typingctx=typing_context,
+                                              targetctx=target_context,
+                                              func=func,
+                                              args=args,
+                                              return_type=return_type,
+                                              flags=flags,
+                                              library=main_library,
+                                              locals={},
+                                              pipeline_class=pipeline_class)
+            except OmnisciUnsupportedError as msg:
+                for m in re.finditer(r'[|]OmnisciUnsupportedError[|](.*?)\n', str(msg), re.S):
+                    print(f'Skipping {fname} `{sig}`: {m.group(0)[25:]}')
```

Comment:
Remove this `print` statement? Or change it by a warning?

Reply:
Changed to warning.